chat/src/main/java/net/md_5/bungee/api/Keybind.java

```diff
@@ -0,0 +1,86 @@
+package net.md_5.bungee.api;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import net.md_5.bungee.api.chat.KeybindComponent;
+
+/**
+ * All possible keybind values used by {@link KeybindComponent}.
+ */
+@RequiredArgsConstructor
+public enum Keybind
```

Comment:
String constants would work, too.
The default key is used in toPlainText() and toLegacyText(). When changing to string constants, what value should be used there?